Kitodo/src/main/java/de/sub/goobi/helper/Helper.java

```diff
@@ -596,4 +596,12 @@ public static String getLastMessage() {
         return compoundMessage;
     }
 
+    /**
+     * Set activeMQReporting.
+     *
+     * @param activeMQReporting as java.util.Map<java.lang.String,java.lang.String>
+     */
+    public static void setActiveMQReporting(Map<String, String> activeMQReporting) {
+        Helper.activeMQReporting = activeMQReporting;
```

Comment:
Instead of `Helper.activeMQReporting = activeMQReporting;` it should be `this.activeMQReporting = activeMQReporting;` or?

Reply:
Helper is a static class so it shouldn't be accessed by instance (this).